src/artofillusion/polymesh/PolyMeshViewer.java

```diff
@@ -357,7 +357,7 @@ private void drawVertices()
             else
                 ref = i;
 
-            Color color = selected[ref] && visible[i] ? selectedVertColor : vertColor;
+            Color color = !selected[ref] && visible[i] ? selectedVertColor : vertColor;
```

Comment:
And this version just inverts the colors. It renders:

- Un-selected, visible vertices in `selectedVertColor`.
- Selected vertices in `vertColor`.
- Unselected *hidden* vertices in `vertColor`.